proto/osmosis/downtime-detector/v1beta1/genesis.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package osmosis.downtimedetector.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v13/x/downtime-detector/types";
+
+// Params holds parameters for the downtime-detector module
+message Params {}
+
+message GenesisDowntimeEntry {
```

Comment:
nit: consider calling it just "DowntimeEntry". Once the module launches, the "Genesis" prefix might feel confusing, especially if we want to make this be configurable.

Reply:
But its not the live state entry format, hence the genesis prefix